.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 - docker
 # Pull container image from Docker Hub
 before_install:
-- docker pull f5devcentral/containthedocs
```

Comment:
This change isn't actually needed -- "latest" is pulled by default when you don't specify a tag.

Reply:
The changes I do think need to be made here are to the requirements.txt file:
```
awscli
git+https://github.com/f5devcentral/f5-sphinx-theme@master \\ change to f5-sphinx-theme>=2.0.2
sphinx>=1.7 \\ this is the minimum version specified in the theme's requirements file
recommonmark 
sphinxjp.themes.basicstrap
cloud_sptheme
```

Reply:
Thanks Jodie, I've made the update!

We've already updated so many repos using "latest'. I'd like keep it this way for consistency. It shouldn't hurt anything?

Reply:
That's fine.